### PR TITLE
[SRVCOM-1162] 1.12.0 Release Notes

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-rn-1-12-0.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* {ServerlessProductName} now uses Knative `kn` CLI 0.18.4.
+* {ServerlessProductName} now uses Knative Kafka 0.18.0.
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
+
+[id="known-issues-1-12-0_{context}"]
+== Known issues
+
+* Using the `eventing.knative.dev/scope: namespace` annotation for the `KafkaChannel` objects is not supported.
```
Comment (matzew) [inline modules/serverless-rn-1-12-0.adoc:24]: +1

**Review by mgencur — CHANGES_REQUESTED** [review]
Looks good overall. Some additions to Fixed Issues requested.

File: modules/serverless-rn-1-12-0.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.18.2.
+* {ServerlessProductName} uses Knative Eventing 0.18.6.
+* {ServerlessProductName} now uses Kourier 0.18.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.18.4.
+* {ServerlessProductName} now uses Knative Kafka 0.18.0.
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
```
Comment (cardil) [inline modules/serverless-rn-1-12-0.adoc:19]: Shouldn't we list all CVE's we are fixing with this release?

**Comment by mgencur** [comment]
Overall, the release notes look good to me now.

File: modules/serverless-rn-1-12-0.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/serverless-rn-1-12-0.adoc:19]: We don't generally, it's listed in the errata already?

**Review by sfortner-RH — APPROVED** [review]
lgtm!

**Pull request merged**